lib/karafka/processing/result.rb

```diff
@@ -18,11 +21,16 @@ def success?
       # Marks state as successful
       def success!
         @success = true
+        # We set cause to false so the previous error that occurred does not leak when error is
```

Comment:
I think that the comment should be above cause

Reply:
yeah, the order was fixed.